src/components/paneler/innhold/personaliaV2/epost.tsx

```diff
@@ -23,9 +23,17 @@ function Epost(props: { epost: OrNothing<PersonaliaEpost> }) {
 
 	return (
 		<Informasjonsbolk header="Epost" {...rest}>
-			<Normaltekst className="innrykk wrap-anywhere">
+			<Normaltekst className="innrykk wrap-anywhere display-flex">
 				{epostAdresse}
-				{epostAdresse && <Kopiknapp kopitekst={epostAdresse} type="epost" />}
+				{epostAdresse && (
+					<CopyToClipboard
+						copyText={epostAdresse}
+						popoverText="Kopiert"
+						popoverPlacement="top"
+						size="xsmall"
+						title="Kopier epost"
```

Comment:
[Ref. Språkrådet](https://www.sprakradet.no/svardatabase/sporsmal-og-svar/e-post-eller-epost):

```suggestion
						title="Kopier e-post"
```

Reply:
Done ✅